tabbycat/options/preferences.py

```diff
@@ -242,6 +242,7 @@ class DrawPullupRestriction(ChoicePreference):
     choices = (
         ('none', _("No restriction")),
         ('least_to_date', _("Choose from teams who have been pulled up the fewest times so far")),
+        ('easy_draw', _("Choose from teams with a low draw strength from previous rounds"))
```

Comment:
Should this say "lowest" rather than just "low"? It's not really clear what "low" means (bottom decile? bottom quartile? bottom half?). I think we're really choosing among those teams that have the minimum draw strength in the bracket, right?

(Also, since this is done by speaks, in most cases this will be a unique team, right?)